pymongo/srv_resolver.py

```diff
@@ -24,7 +24,7 @@
 
 from pymongo.common import CONNECT_TIMEOUT
 from pymongo.errors import ConfigurationError
-
+from random import sample
```

Comment:
Imports from the std lib should go before pymongo imports and before 3rd party (dns) imports.

Reply:
Done.